llvm/test/CodeGen/NVPTX/cluster-dim.ll

```diff
@@ -0,0 +1,17 @@
+; RUN: llc < %s -march=nvptx -mcpu=sm_90 | FileCheck %s
+; RUN: llc < %s -march=nvptx64 -mcpu=sm_90 | FileCheck %s
+; RUN: %if ptxas %{ llc < %s -march=nvptx -mcpu=sm_90 | %ptxas-verify %}
+; RUN: %if ptxas %{ llc < %s -march=nvptx64 -mcpu=sm_90 | %ptxas-verify %}
```

Comment:
Do we need "ptxas-12.0" here, since this is only supported from sm_90 onwards?

Reply:
good point, fixed

Reply:
okay, thanks!